packages/liotad/liotapkg.sh

```diff
@@ -65,5 +65,4 @@ then
 fi
 
 # Echo to named pipe
-echo "Pipe file: $package_messenger_pipe" >&2
```

Comment:
If we are removing this, we should have some response for the user. Something like "Command queued for execution"

Reply:
I am ok with either way. In fact, have you ever seen you linux cmd shows "have put cmd xxx into named pipe"? It seems you are already familiar with current style, so I put it back.

Reply:
In fact, i have never seen a linux command which works on a pipe and doesn't give back the output! :)

Until we start returning the output, don't you think it is best to notify the user that the command is queued?

Reply:
You can check package manager thread directly to see why the current design doesn't give back the output. Anyway, I have put the echo message back.

Reply:
The concern was never my familiarity with the echo or package manager design. It was all to give back a more useful message to the user, which will make sense to him.